modules/tinymce/src/plugins/paste/main/ts/core/Clipboard.ts

```diff
@@ -209,7 +209,8 @@ const isImage = (editor: Editor) => {
 };
 
 const getImagesFromDataTransfer = (editor: Editor, dataTransfer: DataTransfer): File[] => {
-  const items = dataTransfer.items ? Arr.map(Arr.from(dataTransfer.items), (item) => item.getAsFile()) : [];
+  let items = dataTransfer.items ? Arr.map(Arr.from(dataTransfer.items), (item) => item.getAsFile()) : [];
+  items = items.filter((item) => item !== null);
```

Comment:
We'd like to make this small change to your logic to avoid an additional iteration over the array, as well as explicitly checking the `item.kind` before calling `getAsFile()` unnecessarily.

```suggestion
  const items = dataTransfer.items ? Arr.bind(Arr.from(dataTransfer.items), (item) => {
    return item.kind === 'file' ? [ item.getAsFile() ] : [];
  }) : [];
```

Reply:
Tested and validate.